scripts/azureml-assets/azureml/assets/release/asset_publish.py

```diff
@@ -151,6 +191,43 @@ def _str2bool(v: str) -> bool:
             if results.returncode != 0:
                 logger.log_warning(f"Error creating {asset}")
                 failure_list.append(asset)
+            
+        elif asset.type == assets.AssetType.MODEL:
+
+            model_config = asset.model_config_as_object()
+            model_dir = model_prepare(model_config, asset.spec_with_path)
+            # Assemble command
+            cmd = [
+                shutil.which("az"), "ml", asset.type.value, "create",
```

Comment:
@skasturi I think this is where you mentioned we should have favors as per @praveenmathamsetty suggestion.
@haritha-bendapudi can this be incorporated?

Reply:
Also, how we are enforcing semver here?